### ISSUE TITLE
Any way to get the name or IP of the Echo device?

### ISSUE THREAD

**Issue by amera64** [originallyus/node-red-contrib-alexa-local#96]
I see there is a value in the message object for "from:" but the IP address is always the same no matter witch Echo device I speak to.  A name or IP address would be handy when using the Alexa-speak node to direct the speech output to the device that was spoken to.

**Comment by justingriffiths909** [comment]
Agreed. I thought this would solve all my issues and i could use one flow and differentiate by source echo (IP Address).
Is it a bug or a limitation on which device first uses the device?

**Comment by smoofus** [comment]
Getting the echo device name, serial, IP, etc... would be a game changer. It really is the missing link for room based flows.  I just want to be able to say "Turn on the TV" and have the TV in the room with the echo that received the command turn on. +1

**Comment by derSebbl** [comment]
There is a fork of this repository in which the ip adress is available:
https://github.com/mabunixda/node-red-contrib-alexa-home
(Gen 3 detection also works with a workaround) (:
